server/src/test/java/org/apache/druid/client/DirectDruidClientTest.java

```diff
@@ -143,33 +147,33 @@ public void testRun() throws Exception
     SettableFuture<InputStream> futureResult = SettableFuture.create();
     Capture<Request> capturedRequest = EasyMock.newCapture();
     EasyMock.expect(
-        httpClient.go(
-            EasyMock.capture(capturedRequest),
-            EasyMock.<HttpResponseHandler>anyObject(),
-            EasyMock.anyObject(Duration.class)
-        )
-    )
+                httpClient.go(
+                    EasyMock.capture(capturedRequest),
+                    EasyMock.<HttpResponseHandler>anyObject(),
+                    EasyMock.anyObject(Duration.class)
+                )
+            )
```

Comment:
This re-formatting shouldn't have happened. Are you using a different checkstyle?